src/Helpers/LeanCode.UserIdExtractors/LeanCode.UserIdExtractors.csproj

```diff
@@ -0,0 +1,6 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Do we need to pull the entire AspNetCore here? This looks like this package could get away with just some `DependencyInjection.Abstractions` reference.